io/net/iperf_test.py

```diff
@@ -98,10 +99,12 @@ def setUp(self):
         archive.extract(tarball, self.iperf)
         self.version = os.path.basename(tarball.split('.tar')[0])
         self.iperf_dir = os.path.join(self.iperf, self.version)
-        destination = "%s:/tmp" % self.peer_ip
-        output = self.session.copy_files(self.iperf_dir, destination,
-                                         recursive=True)
-        if not output:
+        out = self.session.get_raw_ssh_command('ls')
```

Comment:
@vaishnavibhat why you changed this like before .? already handle this as per utils.